murdock/templates/comment.md.j2

```diff
@@ -38,6 +38,15 @@
 
 </details>
 {% endif %}
+{% if job.config.pr.comment_artifacts %}
+
+#### Artifacts
+{% for artifact in job.config.pr.comment_artifacts %}
+{% if artifact.name in job.artifacts %}
+- [{{ artifact.readable_name }}]({{ job.http_dir }}/{{ artifact.name }})
```

Comment:
The `job.http_dir` is not enough, it doesn't contain the base url. You probably have to extends the context of the template with `GLOBAL_CONFIG.base_url`:

https://github.com/murdock-ng/murdock/blob/main/murdock/github.py#L73

woud become:

```python
context = {"job": job, "base_url": GLOBAL_CONFIG.base_url}
```

And then this line:
```suggestion
- [{{ artifact.readable_name }}]({{ base_url }}/{{ job.http_dir }}/{{ artifact.name }})
```

Reply:
Mhhh I was copying this from somewhere where the URL is used as well. Maybe I just need to fix the template ;-).

Reply:
Ah no, you are right!